### ISSUE TITLE
Warning: uXRCE-DDS now supports services

### ISSUE THREAD

**Issue by beniaminopozzan** [PX4/PX4-user_guide#2876]
After https://github.com/PX4/PX4-Autopilot/pull/22313 you have to use ROS2 services to send VehicleCommands.
Doc update is required.

Example of usage is here: https://github.com/PX4/px4_ros_com/pull/207

**Comment by hamishwillee** [comment]
That's cool. So the beginning of a proper interface. Is this associated with the SDK?
When this is done, please add a release note too.

**Comment by beniaminopozzan** [comment]
EDIT: `vehicle_command` has been added again in https://github.com/PX4/PX4-Autopilot/pull/20707 
Therefore, no change in the interface on `main`. Docs is still valid, just need to be update with the new feature

**Comment by bkueng** [comment]
Yes we can switch the library to use the service later on, if we know the service works reliably.

**Comment by hamishwillee** [comment]
OK. I'll look forward to seeing docs for this :-).

**Comment by hamishwillee** [comment]
@beniaminopozzan I'm currently having a bit of trouble with https://github.com/PX4/PX4-user_guide/pull/3465 because we haven't documented services or service topics. I.e. the doc makes sense in and of itself, as long as you believe in this "magic fairy thing called a service".

Would it be possible for you to do so, or help update that doc as it iterates?

**Comment by beniaminopozzan** [comment]
@hamishwillee you are right, I'll see to have a PR open in the next days

![Image](https://github.com/user-attachments/assets/48746f8a-58be-4f97-81e0-7cf0669437b5)

Yes, it is long overdue

**Comment by hamishwillee** [comment]
Thanks very much!
